security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,6 +12,22 @@ These release notes track the development of {cert-manager-operator}.
 
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
+[id="cert-manager-operator-release-notes-1.11.4"]
+== Release notes for {cert-manager-operator} 1.11.4
+
+Issued: 2023-07-26
+
+The following advisory is available for the {cert-manager-operator} 1.11.4:
+
+* link:https://access.redhat.com/errata/RHEA-2023:4081[RHEA-2023:4081]
+
+For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.11/[cert-manager project release notes for v1.11]. The golang version is updated to the version `1.19.10` to fix Common Vulnerabilities and Exposures (CVEs).
+
+[id="cert-manager-operator-1.11.4-bug-fixes"]
+=== Bug fixes
+
+* Previously, the {cert-manager-operator} did not allow installing the older versions of the {cert-manager-operator}. Now, you can install the {cert-manager-operator} older versions by selecting the *Update approval* strategy as *Manual* in the web console. For more information, see xref:../../security/cert_manager_operator/cert-manager-operator-install.adoc#cert-manager-operator-install[Installing the {cert-manager-operator}]. (link:https://issues.redhat.com/browse/OCPBUGS-16393[*OCPBUGS-16393*])
```

Comment:
The preview page looks like `Now, you can install the cert-manager Operator for Red Hat OpenShift older versions`. The long name of the operator always will bring such issue :), that is, it can be always intuitively read as ambiguously as `the cert-manager Operator for "Red Hat OpenShift older versions"` by others.
```suggestion
* Previously, the {cert-manager-operator} did not allow installing the older versions of the {cert-manager-operator}. Now, you can install older versions of the {cert-manager-operator} by selecting the *Update approval* strategy as *Manual* in the web console. For more information, see xref:../../security/cert_manager_operator/cert-manager-operator-install.adoc#cert-manager-operator-install[Installing the {cert-manager-operator}]. (link:https://issues.redhat.com/browse/OCPBUGS-16393[*OCPBUGS-16393*])
```

Reply:
+1 to `older versions of the {cert-manager-operator}`.

Reply:
Addressed. 
@xingxingxia I added one more line - `You can install older versions of the {cert-manager-operator} by using the command-line interface (CLI) in {product-title} 4.13.` I didn't mention 4.12 because the PR is aiming for 4.13+ versions.

Reply:
Not sure how the aim is decided, but 1.11.4 supports 4.12, the bug fix supports 4.12 too.

Reply:
@xingxingxia Okay, in that case, the PR should target 4.12+ versions. I updated the PR. Thanks for pointing that out.